TestAssets/TestPackages/dotnet-dependency-context-test/project.json

```diff
@@ -1,5 +1,5 @@
 {
-  "version": "1.0.0-preview1-*",
+  "version": "1.0.0-preview2-*",
```

Comment:
I don't understand why these test assets need to be versioned. We don't actually ship them.  Can they just use "1.0.0-*"?